### DISCUSSION TITLE
Core not found

### DISCUSSION THREAD

**Discussion by RockinRowl** [Squirrel/Squirrel.Windows#1782] [Q&A] [answered]
Hi All, I downloaded the code but after opening in Visual Studio, Core is not available.
Why is Core not included in the source code?

![image](https://user-images.githubusercontent.com/61777044/150641427-2d84b332-43dd-421a-b6ac-8606d69d5f5e.png)

**Comment by robmen** [comment, marked as answer]
It comes from a submodule. So: `git submodule update --init --recursive`

**Reply by RockinRowl** [reply]
Thanks for the reply @robmen  but I don't know how or where to run that command, is there a link where I can download it?

**Reply by robmen** [reply]
There are some great tutorials about git. [Here's one for example](https://git-scm.com/).

**Reply by RockinRowl** [reply]
Thanks @robmen, I was able to get it, I just followed this quick tutorial on Git. 
https://www.w3schools.com/git/default.asp?remote=github

Then run the command.

Cheers.
